frontend/components/Header.tsx

```diff
@@ -49,7 +52,7 @@ export const Header: FC = () => {
       {isLoggedIn ? (
         <div className="flex bg-blue-400">
           <div className="cursor-pointer" onClick={openMenu}>
-            <Avatar imageUrl="" />
+            <Avatar imageUrl={user?.Image || "/images/common/mystery.svg"} />
```

Comment:
Using the following would ensure the existence of the file (gives a TS error if the file doesn't exists):
```
import mysteryImg from '../public/images/common/mystery.svg`
...
<Avatar imageUrl={user?.Image || mysteryImg} />
```

Reply:
I added next-images now and converted all images to being imported.

Nice hint 👍

Reply:
I really like this improvement!